components/kyma-environment-broker/internal/process/provisioning/cls_provision_int_test.go

```diff
@@ -74,21 +74,32 @@ func TestClsProvisionSteps(t *testing.T) {
 	offeringStep := NewClsOfferingStep(clsConfig, repo)
 
 	clsClient := cls.NewClient(clsConfig, log)
-	clsProvisioner := cls.NewProvisioner(db.CLSInstances(), clsClient, log)
-	provisionStep := NewClsProvisionStep(clsConfig, clsProvisioner, repo)
+	//clsProvisioner := cls.NewProvisioner(db.CLSInstances(), clsClient, log)
+	//provisioningStep := NewClsProvisioningStep(clsConfig, clsProvisioner, repo)
```

Comment:
why did you commented out provisioning step?

Reply:
fixed.